experimental/ExteriorAlgebra/ExteriorAlgebra_naive.jl

```diff
@@ -0,0 +1,95 @@
+export exterior_algebra_naive;  # constructor
+export ContainsDuplicates;   # export just for developing/testing
+
+# Auxiliary function -- worth keeping?
+
+function ContainsDuplicates(L)  # L must be indexable (1:n), and its elems must be sortable
+  n = length(L);
+  if (n < 2)  return false;  end;
+  if (n == 2)  return (L[1] == L[2]);  end;
+  sorted = sort(L);
+  for i=1:n-1  if (sorted[i] == sorted[i+1])  return true;  end;  end;
+  return false;
+end;
+
+
+# exterior_algebra constructor:
+#  args should be underlying CoeffRing and number of indets
+#  optional keyword arg: indet_prefix (must be roman-alphabetic"
+
+
+# Returns 2 components: ExtAlg, list of the gens/variables in order (e1,..,en)
+# [[depends on what pbw_algebra returns!]]
+@doc Markdown.doc"""
+    exterior_algebra_naive(CoeffRing, NumVars)
```

Comment:
```suggestion
    exterior_algebra_naive(coeff_ring::Ring, num_vars::Int)
```
Why not annotate the types? Also, it might be necessary to match the docstring namings with the actual code below for the documenter to find it. But there I'm not sure.

Reply:
Thanks!  Yes, I can annotate the formal parameters in the docstring.
At the moment I have a (mild) preference for CamelCase for **non-exported** parts of the code -- simply because that is what I am used to.  If there are objections, I can adjust my preferred style.  If you have a persuasive argument why I should change my style, let me know... I'm willing to listen!